app/models/store.rb

```diff
@@ -0,0 +1,31 @@
+class Store < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.